packages/nx/src/plugins/js/lock-file/pnpm-parser.ts

```diff
@@ -40,7 +42,9 @@ function addNodes(
 
   Object.entries(data.packages).forEach(([key, snapshot]) => {
     const packageName = findPackageName(key, snapshot, data);
-    const version = findVersion(key, packageName).split('_')[0];
+    const version = findVersion(key, packageName).split(
+      MATCH_ADDITIONAL_VERSION_INFO
```

Comment:
Can this be a ternary based on the condition of the Lockfile version?

v5 = '_'
v6 = '('